addons/mail/static/src/components/rtc_call_viewer/rtc_call_viewer.xml

```diff
@@ -51,15 +51,15 @@
 
             <!-- Dialogs -->
             <t t-if="messaging.userSetting.rtcConfigurationMenu.isOpen">
-                <Dialog size="'small'" title="'Settings'" t-on-dialog-closed="rtcCallViewer.onRtcSettingsDialogClosed">
+                <Dialog size="'small'" title="'Settings'" onClosed="rtcCallViewer.onRtcSettingsDialogClosed">
                     <RtcConfigurationMenu/>
                     <t t-set-slot="buttons">
                         <!-- Explicit No buttons -->
                     </t>
                 </Dialog>
             </t>
             <t t-if="rtcCallViewer and rtcCallViewer.rtcLayoutMenu">
-                <Dialog size="'small'" title="'Change Layout'" t-on-dialog-closed="rtcCallViewer.onLayoutSettingsDialogClosed">
+                <Dialog size="'small'" title="'Change Layout'" onClosed="rtcCallViewer.onLayoutSettingsDialogClosed">
```

Comment:
I'm not sure if the comment is on the correct line, but when changing layout, it should close the dialog automatically, and it is no longer the case.

Reply:
Also I just saw it and it's unrelated, but this title is currently not translatable as it is encoded as a string.

Reply:
Regarding auto-close of dialog, it was caused by discuss code manually triggering the `dialog-closed` event.
https://github.com/odoo/odoo/blob/master/addons/mail/static/src/models/rtc_layout_menu/rtc_layout_menu.js#L46

I fixed the issue by ensuring `OwlDialog` calls `onClosed` when it intercepts `dialog-closed` event from below (or self).

I didn't have to make the same thing for `Popover` because it was only used internally by `Popover`.